tools/packaging/kernel/build-kernel.sh

```diff
@@ -553,7 +558,8 @@ main() {
 			esac
 		elif [[ "${conf_guest}" != "" ]]; then
 			#If specifying a tag for kernel_version, must be formatted version-like to avoid unintended parsing issues
-			kernel_version=$(get_from_kata_deps "assets.kernel.${conf_guest}.tag")
+			kernel_version=$(get_from_kata_deps "assets.kernel.${conf_guest}.version" 2>/dev/null || true)
+			[ -z "${kernel_version}" ] && kernel_version=$(get_from_kata_deps "assets.kernel.${conf_guest}.tag")
```

Comment:
Avoid the use of `A && B` because that expression can return non-zero status. Instead use `! A || B` which always return zero (ah, unless `B` return fails.. which is not the case here).

Reply:
done